tools/release/changelog.config.js

```diff
@@ -1,7 +1,7 @@
 const {execSync} = require('child_process');
 
 const latestReleaseTag = execSync(
-  `git tag --sort=creatordate |  grep -E '(^alpha-.*)|(^v.*)' | tail -1`,
+  `git tag --sort=creatordate |  grep -E '/(?!(.*(atb|nfk).*))((alpha-.*)|(v.*))' | tail -1`,
```

Comment:
Looks good. Maybe not strictly necessary to check if there are characters before? Or is the thought to support things like `realease/` prefix?

Reply:
The thought is to support anything really, just including an org-check, in case the naming of the releases would change in the future. Are you referring to the removal of the `^`-s when talking about checking characters before?